quodlibet/quodlibet/ext/events/randomalbum.py

```diff
@@ -233,3 +233,10 @@ def unpause(self):
             app.player.next()
         except AttributeError:
             app.player.paused = True
+
+    def disabled_for_browser(self, browser):
+        if not browser.can_filter_albums():
```

Comment:
Oh actually the fact that you said "return" should have informed me as to what you meant.. I'll make that change.